packages/client/src/declarations/submissionMiddleware.ts

```diff
@@ -187,6 +187,7 @@ export const submissionMiddleware: Middleware<{}, IStoreState> =
       graphqlPayload.registration.correction =
         declaration.data.registration.correction ?? {}
       graphqlPayload.registration.correction.values = changedValues
+      delete graphqlPayload.registration.certificates
```

Comment:
@Nil20 Could you please add a comment explaining why we are doing this?

Reply:
so if we're submitting a correction, we remove certificate properties here, what is the reason for it though?

Reply:
The correction request payload doesn't have any `certificates` properties in it.
https://github.com/opencrvs/opencrvs-core/blob/c19bf79ee14d178ee6e22fb596ecd76ef21ca369/packages/workflow/src/records/validations.ts#L76